syntax = "proto2";
import "game_commands.proto";
message Command_RollDie {
    extend GameCommand {
        optional Command_RollDie ext = 1005;
    }
    optional uint32 sides = 1;
    optional uint32 dice_to_roll = 2;
}
